### PR TITLE
add additional info on pv workflow

### PR DESCRIPTION
**Pull request by kflemin**
### Pull Request Description

[description here]

### Checklist (Delete lines that don't apply)

- [ ] Documentation has been modified appropriately
- [ ] An ISSUE has been created that this is addressing. Issues will get added to the Change Log when the change_log.rb script is run
- [ ] This branch is up-to-date with develop

### CODE REVIEW

File: workflows/photovoltaic.md [hunk 1]
```diff
@@ -15,7 +15,15 @@ Community Photovoltaic features can be added to the GeoJSON Feature File as a di
...
+
+Alternatively, ground mount photovoltaics can be considered in the REopt feature optimization without defining district system features and associating them with a building in the GeoJSON file. If no features are defined in the GeoJSON file, REopt return the optimal amount of ground mount PV without any area limitations for each feature that was run with an [assumptions file](#assumptions-files) that contained a PV definition with location = `ground`. More realistically, there will be area limitations and, therefore, district systems defined in the GeoJSON files for those building features where ground mount PV should be considered.  An additional refinement would be to specify different assumptions files for features with ground mount PV and those without. Feature-by-feature assumptions file assignment is done in the scenario CSV file for a particular [reopt-enabled](../getting_started/getting_started.md#step2) scenario. 
+
+### Ground Mount Photovoltaic Features and REopt Assumptions Files
+
+While ground mount photovoltaic features can be
```
Comment (tanushree04) [inline workflows/photovoltaic.md:26]: @kflemin We can remove this heading I think.

File: workflows/photovoltaic.md [hunk 2]
```diff
@@ -15,7 +15,15 @@ Community Photovoltaic features can be added to the GeoJSON Feature File as a di
...
+
+Alternatively, ground mount photovoltaics can be considered in the REopt feature optimization without defining district system features and associating them with a building in the GeoJSON file. If no features are defined in the GeoJSON file, REopt return the optimal amount of ground mount PV without any area limitations for each feature that was run with an [assumptions file](#assumptions-files) that contained a PV definition with location = `ground`. More realistically, there will be area limitations and, therefore, district systems defined in the GeoJSON files for those building features where ground mount PV should be considered.  An additional refinement would be to specify different assumptions files for features with ground mount PV and those without. Feature-by-feature assumptions file assignment is done in the scenario CSV file for a particular [reopt-enabled](../getting_started/getting_started.md#step2) scenario.
```
Comment (tanushree04) [inline workflows/photovoltaic.md:22]: Can we say return the optimal amount of ground mount PV **with an upper limit of area equivalent to the site are**a for each feature that was run with an [assumptions file](#assumptions-files) that contained a PV definition with location = `ground`.  More realistically, there will **be a more specific area limitation based on the area available for Ground Mount PV associated with the Feature** and, therefore, ....

File: workflows/photovoltaic.md (hunk 1 above)
Comment (tanushree04) [reply workflows/photovoltaic.md:26]: I think we can mention `gcr` is 1 for Rooftop PV as well
https://github.com/urbanopt/urbanopt.github.io/pull/142/files#diff-57ba422b5d77918e3b49f29da2662e3d43b2995773e30137ed2b7a0403ec2e87L22

File: workflows/photovoltaic.md (hunk 2 above)
Comment (kflemin) [reply workflows/photovoltaic.md]: thanks! this should be fixed

File: workflows/photovoltaic.md (hunk 1 above)
Comment (kflemin) [reply workflows/photovoltaic.md]: oops, i removed it

**Pull request merged**